test/test_namedtensor.py

```diff
@@ -1241,26 +1245,26 @@ def kthvalue_wrapper(tensor, *args, **kwargs):
         ])
 
         tests = [
-            Case(torch.sum, True, True, True, True, None),
-            Case(torch.prod, True, False, True, True, None),
-            Case(torch.mean, True, True, True, True, None),
-            Case(torch.var, True, True, True, True, None),
-            Case(torch.std, True, True, True, True, None),
-            Case(torch.std_mean, True, True, False, True, None),
-            Case(torch.var_mean, True, True, False, True, None),
-            Case(torch.min, True, False, True, True, values_and_indices),
-            Case(torch.max, True, False, True, True, values_and_indices),
-            Case(torch.unbind, False, False, False, False, None),
-            Case(torch.logsumexp, False, True, True, True, None),
-            Case(torch.mode, False, False, True, True, values_and_indices),
-            Case(kthvalue_wrapper, False, False, True, True, values_and_indices),
-            Case(torch.median, True, False, True, True, values_and_indices),
-            Case(torch.nanmedian, True, False, True, True, values_and_indices),
+            Case(torch.sum, False, True, True, True, True, None),
```

Comment:
fyi @heitorschueroff; it'll be nice to revise this test so we don't deal with all these unnamed tuples